src/pages/Research.jsx

```diff
@@ -54,86 +54,49 @@ export default function Research() {
 function ResearchExplorer() {
   const displayCategory = useDisplayCategory();
   const [searchParams, setSearchParams] = useSearchParams();
-  const onSearch = (search) => {
-    if (!search) {
-      setSearchParams({});
-    } else {
-      setSearchParams({ search });
-    }
-    window.scrollTo(0, 0);
-  };
 
+  // Extract the country ID from the URL pathname
   const extractCountryIdFromPathname = () => {
     const pathSegments = window.location.pathname.split("/").filter(Boolean);
     if (pathSegments.length > 0) {
       return pathSegments[0];
     }
   };
 
+  // Determine initial locations based on the country ID
   const initialLocations = locationTags.filter(
     (location) =>
       location === extractCountryIdFromPathname() ||
       location.startsWith(extractCountryIdFromPathname() + "-") ||
       location === "global",
   );
 
+  // Initialize filter states with the appropriate values
   const [filteredTopics, setFilteredTopics] = useState(
     searchParams.get("topics")?.split(",") || topicTags,
```

Comment:
```suggestion
    searchParams.get("topics")?.split(",") || [],
```
Here and below, I'm using an empty array because I think this is our desired default - for everything other than locations, it should be no filters selected instead of all selected, even though the outcome is nearly the same for the user.